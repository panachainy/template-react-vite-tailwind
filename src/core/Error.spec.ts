import { describe, expect, it } from 'vitest'
import ErrorPage from './Error'

describe('ErrorPage component', () => {
  it('should match the snapshot', () => {
    const result = ErrorPage({})
    expect(result).toMatchSnapshot()
  })
})
