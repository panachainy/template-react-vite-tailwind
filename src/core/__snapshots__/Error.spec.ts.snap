// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ErrorPage component > should match the snapshot 1`] = `
<div>
  <h1>
    Oops!
  </h1>
  <p>
    Sorry, an unexpected error has occurred.
  </p>
</div>
`;
